### PR TITLE
[Term Entry] CSS Filter Function: grayscale() 

### PR DESCRIPTION
**Pull request by giliopoulosaz**
### Description

- Added a image/media .png for grayscale()
- Updated grayscale().md
- Closes Issue CSS [Edit] CSS Filter Functions: grayscale() #3966

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by dakshdeepHERE** [comment]
@giliopoulosaz looks good to me 🚀

**Review by dakshdeepHERE — APPROVED** [review]
Looks good for second review 🚀

File: media/css-filterfunction-grayscale-20%.png
Comment (SSwiniarski) [inline media/css-filterfunction-grayscale-20%.png:1]: This particular image doesn't show a noticeable difference when greyscale is applied. Can we use an image where the filter has a noticeable effect?

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
I have one comment about the image you're using.

File: media/css-filterfunction-grayscale-20%.png
Comment (giliopoulosaz) [reply media/css-filterfunction-grayscale-20%.png:1]: I can increase the the filter to 50% or 60% on the current image to make it more noticeable, but I wanted to keep the document consistent with the examples above that has 20%. But yes I can change the image to another to see the changes at 20%.

**Comment by giliopoulosaz** [comment]
@SSwiniarski I updated the image to one that shows a more noticeable difference with the grayscale at 20%.

**Review by SSwiniarski — APPROVED** [review]
@giliopoulosaz looks good.

**Pull request merged**